python/ray/data/_internal/execution/resource_manager.py

```diff
@@ -696,14 +709,8 @@ def update_usages(self):
         remaining_shared = self._total_shared
         for op in eligible_ops:
             # Calculate the memory usage of the operator.
-            op_mem_usage = 0
-            # Add the memory usage of the operator itself,
-            # excluding `_reserved_for_op_outputs`.
-            op_mem_usage += self._resource_manager._mem_op_internal[op]
-            # Add the portion of op outputs usage that has
-            # exceeded `_reserved_for_op_outputs`.
-            op_outputs_usage = self._get_op_outputs_usage_with_downstream(op)
-            op_mem_usage += max(op_outputs_usage - self._reserved_for_op_outputs[op], 0)
+            op_mem_usage = self._resource_manager._mem_op_internal[op]
+            op_mem_usage += self._get_op_outputs_usage_with_downstream(op)
```

Comment:
Previously, `_op_budget` represented the budget that the operator could use for scheduling new tasks, which is the total budget excluding the unused memory reserved for outputs. Now, `_op_budget` just represents the total budget for the operator, including the portion reserved for outputs.